app/models/project_type.rb

```diff
@@ -42,7 +42,8 @@ class ProjectType < ActiveRecord::Base
 
   has_many :available_project_statuses, class_name:  'AvailableProjectStatus',
                                         foreign_key: 'project_type_id',
-                                        dependent: :destroy
+                                        dependent: :destroy,
+                                        validate: false
```

Comment:
This was done in a commit marked as [WIP]. Do you intend to change this back @myabc?